dipy/io/image.py

```diff
@@ -24,3 +25,38 @@ def load_nifti(fname, return_img=False, return_voxsize=False,
 def save_nifti(fname, data, affine, hdr=None):
     result_img = nib.Nifti1Image(data, affine, header=hdr)
     result_img.to_filename(fname)
+
+
+def save_affine_matrix(fname, affine):
+    """Save Affive matrix.
+
+    Parameters
+    ----------
+    fname : string
+        File name to save the affine matrix.
+    affine : numpy array
+        The object containing the affine matrix.
+
+    """
+    np.savetxt(fname, affine)
```

Comment:
Why do we need a wrapper for `np.savetxt`? Couldn't we just use that directly?

Reply:
Even if I have the same opinion, it is always good to have an explicit function name for the end-user. Currently, they have no "explicit official" way to save an affine matrix. I do not have a strict opinion about that, I can remove it. What do you think?